packages/schematics/src/collection/ngrx/rules/add-exports-barrel.ts

```diff
@@ -31,11 +32,20 @@ export function addExportsToBarrel(options: Schema): Rule {
           true
         );
 
-        // Public API for the feature interfaces, selectors
+        // Public API for the feature interfaces, selectors, and facade
         const { fileName } = names(options.name);
         const statePath = `./lib/${options.directory}/${fileName}`;
 
         insert(host, indexFilePath, [
+          // Only add the Facade to the barrel.
+          // This facade is the ONLY public access to the managed state.
```

Comment:
I don't think this comment reflects the implementation (e.g., You haven't removed the reexported actions etc. )